tools/package_gcsfuse_docker/Dockerfile

```diff
@@ -60,6 +60,10 @@ RUN mv ${GCSFUSE_BIN}/DEBIAN/copyright ${GCSFUSE_DOC} &&  \
 # Update gcsfuse version in changelog and control file
 RUN sed -i "1s/.*/gcsfuse (${GCSFUSE_VERSION}) stable; urgency=medium/" ${GCSFUSE_DOC}/changelog && \
     sed -i "1s/.*/Version: ${GCSFUSE_VERSION}/" ${GCSFUSE_BIN}/DEBIAN/control
+
+# Update ARCHITECTURE in control file.
+RUN sed -i "6s/.*/Architecture: ${ARCHITECTURE}/" ${GCSFUSE_BIN}/DEBIAN/control
```

Comment:
I think we should add a check here to ensure that the passed architecture is arm64 or amd64 only.

Reply:
I don't think so it is require, what ever architecture user will pass it will update accordingly. In future if we support i386 architecture, we can pass that also.

Reply:
what if someone makes a typo in the passed architecture? 
Although the process is fairly automated now, but, i still think it is still good to add a check.